test/e2e/allocator_test.go

```diff
@@ -110,9 +107,6 @@ func TestAllocatorCrossNamespace(t *testing.T) {
 	requestURL := fmt.Sprintf(allocatorReqURLFmt, ip, port)
 	tlsCA := refreshAllocatorTLSCerts(t, ip)
 
-	// Create namespaces A and B
-	namespaceA := framework.Namespace // let's reuse an existing one
```

Comment:
The default namespace always has a game server. This test verifies that the game server is allocated from namespace B, because namespace A doesn't have a game server. If you want to change the namespace to default, you should also verify the gameserver origin. Otherwise, change the namespace creation logic to copy over the allocator-client.default cert. I prefer the latter.

Reply:
Done!